JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts

```diff
@@ -20,13 +21,19 @@ if (!/localhost/.test(document.location.host)) {
 export class AppComponent {
   dataSource: DataSource;
 
+  dinnerTime;
+
+  holidays;
+
   currentDate = new Date(2021, 3, 27);
 
   views = ['workWeek', 'month'];
 
   currentView = this.views[0];
 
   constructor(public dataService: DataService) {
+    this.dinnerTime = dataService.getDinnerTime();
+    this.holidays = dataService.getHolidays();
     this.dataSource = new DataSource({
       store: dataService.getData(),
     });
```

Comment:
Let's get rid of useless constructor here:

```suggestion
  dataSource = new DataSource({
      store: this.dataService.getData(),
  });

  dinnerTime = this.dataService.getDinnerTime();

  holidays = this.dataService.getHolidays();

  currentDate = new Date(2021, 3, 27);

  views = ['workWeek', 'month'];

  currentView = this.views[0];

  constructor(private dataService: DataService) {}
```